src/schema/v2/types/recentlySoldArtworksConnection.ts

```diff
@@ -0,0 +1,45 @@
+import { ResolverContext } from "types/graphql"
```

Comment:
Can the following new files be moved to a `v2/types/targetSupply` folder?

Reply:
Yup! I'll update this PR.

Reply:
Updated!